.travis.yml

```diff
@@ -17,6 +17,13 @@ env:
     - PRODUCT_TEST_GROUP=5
     - PRODUCT_TEST_GROUP=6
     - PRODUCT_TEST_GROUP=7
+    - PRODUCT_TEST_GROUP=8
+    - PRODUCT_TEST_GROUP=9
+    - PRODUCT_TEST_GROUP=10
+    - PRODUCT_TEST_GROUP=11
+    - PRODUCT_TEST_GROUP=12
+    - PRODUCT_TEST_GROUP=13
+    - PRODUCT_TEST_GROUP=14
```

Comment:
travis not only fails because of timeouts. Travis has low amount of resources. There are tests which spin up a docker cluster which consists of 4 docker containers where each tries to start presto. In such case travis kills some processes as the memory usage is exceeding travis capacity. I think this is the most serious issue. 

Moreover, when a process is killed, then presto-admin waits until timeout to reports that presto-server didn't show up. Thus making tests last longer.